samples/expo/app/(tabs)/index.tsx

```diff
@@ -17,6 +17,7 @@ export default function TabOneScreen() {
   const { currentlyRunning } = useUpdates();
   return (
     <View style={styles.container}>
+      {/* <Image source={require('../..//hi.gif')} style={{ width: 100, height: 100 }} /> */}
```

Comment:
q: Why the comment?

Reply:
Just an oversight, I'll remove it, thank you.